matchbox_signaling/examples/client_server.rs

```diff
@@ -0,0 +1,34 @@
+use matchbox_signaling::SignalingServer;
+use std::net::Ipv4Addr;
+use tracing::info;
+
+#[tokio::main]
+async fn main() -> Result<(), matchbox_signaling::Error> {
+    setup_logging();
+
+    let server = SignalingServer::client_server_builder((Ipv4Addr::UNSPECIFIED, 3536))
+        .on_connection_request(|cxn| {
+            info!("Connecting: {cxn:?}");
```

Comment:
I also find the `cxn` abbreviation a bit weird. Better to spell it out, since the lambda doesn't include a type annotation?